aopy/data/bmi3d.py

```diff
@@ -641,18 +669,57 @@ def get_interp_task_data(exp_data, exp_metadata, datatype='cursor', samplerate=1
         clock = exp_data['clock']['timestamp_bmi3d']
 
     # Fetch the relevant BMI3D data
-    if datatype == 'hand':
-        data_cycles = exp_data['clean_hand_position'] # 3d hand position (optitrack coords: x,y,z) on each bmi3d cycle
+    if datatype in ['hand', 'user_raw', 'manual_input']:
+        warnings.warn("Raw hand position is not recommended for analysis. Use 'manual_input' instead for 3D world coordinate inputs.")
```

Comment:
Wouldn't it be use "user_world" instead?

Reply:
oh yeah